lib/cli/commands-schema/no-service.js

```diff
@@ -159,6 +159,16 @@ commands.set('login', {
   usage: 'Login or sign up for Serverless',
   serviceDependencyMode: 'optional',
```

Comment:
Should we remove the service dependency mode at this point? Or it should still take an effect if it's run in the context of Console service?

Reply:
It's still unfortunately needed, as we have triage on wether to ask user to which service login or not, and we need to have configuration for that

Reply:
thanks for clarification 👍